packages/shared/src/hooks/usePostById.ts

```diff
@@ -129,7 +131,9 @@ const usePostById = ({ id, options = {} }: UsePostByIdProps): UsePostById => {
     staleTime: StaleTime.Default,
     enabled: !!id && tokenRefreshed,
   });
-  const post = postById || (options?.initialData as PostData);
+  const post = isPlus
+    ? postById
+    : postById || (options?.initialData as PostData);
```

Comment:
Think we should revert this https://github.com/dailydotdev/apps/pull/3930/files#r1872952334

Reply:
The problem is that when using the SSG, its value gets set as initial data in the useSmartTitle query state, so you'll see the original title even if you have clickbait shield enabled

Reply:
Maybe avoid using initialData, initialData is more like a default in useState, so its tricky because it only takes the first ever value, here we need to conditionally set one or the other

Reply:
I think I got it to work, opening separate PR for it to test

Reply:
Reverted it in #3932